### ISSUE TITLE
Arbiter master / spare, which is which?

### ISSUE THREAD

**Issue by mohierf** [Alignak-monitoring/alignak#609]
**When a configuration has multiple arbiters, how to determine which one is the master arbiter?**

 When the arbiter starts, it loads the monitoring configuration and gets the list of the known arbiters in this configuration. From this list, it tries to determine which arbiter in the list is the master one. **But...** the arbiter daemon, which is a daemon do not know its own name, thus it is not really able to choose which arbiter in the list is its own configuration.

 So, what happens: 
 - each arbiter daemon process do not know which arbiter type it is ... master or spare
 - each arbiter loads the configuration and finds an arbiter list
 - each arbiter gets stuck because it can not find its own configuration in the list ...

 Formerly, the arbiter received its name from the command line parameters but this feature has been removed several months ago in benefit of a configuration name that can be provided by the command line :/

 There are several solutions to this problem: 
 1. restore a command line parameter to set the arbiter name
    * This solution is only an arbiter solution that does not suit other daemons. But it is not necessary ...*

 1. create a command line parameter to indicate that the running daemon is a spare one
    * This solution may be used for all the daemons and it may replace the `spare` parameter in the daemon configuration file. May be interesting ...* :+1: for me on this solution

 1. create a `spare` parameter in the `arbiter.ini` file
    * The Daemon class still has an unused spare parameter, why not using this to indicate in the daemon ini configuration file that it is a spare daemon?*

-----
Arbiter is a specific daemon for which command line interface is different from all other daemons (see hereunder).

```
Arbiter only:
        "-a", "--arbiter": Monitored configuration file(s), (multiple -a can be used, and they will be concatenated. ')
        "-V", "--verify-config": Verify configuration file(s) and exit
        "-n", "--config-name": Use the name of the arbiter defined in the configuration files (default Default-arbiter)")

All daemons:
    '-c', '--config': Daemon configuration file
    '-d', '--daemon': Run as a daemon
    '-r', '--replace': Replace previous running daemon
    '-f', '--debugfile': File to dump debug logs

-----

**Comment by mohierf** [comment]
Note that there is another solution I did not mentionned in the former comment. Check tha t the current host name / address is the one that is defined for the current arbiter daemon, but this solution is the more restrictive one :pensive:

**Comment by mohierf** [comment]
After some more tests, using the `-n` command line parameter is ok for the arbiter to find its own configuration in the monitoring configuration. Once found this configuration indicates if it is a master or spare one ... but both master and spare arbiter need to parse the configuration. It is not an optimal solution ...

**Comment by mohierf** [comment]
Managed with #626

**Comment by mohierf** [comment]
Closed thanks to #669
